scripts/docker/debian/stretch/Dockerfile.minimal

```diff
@@ -5,7 +5,7 @@ ENV LANGUAGE C.UTF-8
 ENV LC_ALL C.UTF-8
 
 RUN apt-get update && apt-get -y install \
-    cmake git build-essential curl
+    cmake git build-essential curl file
```

Comment:
This kind of defeats the purpose of having a minimal test environment.
The packages listed here are taken straight from COMPILE.md and should be enough to build + run tests on the 'core'.

To make this change make sense we either need to
  * add `file` as a dependency to COMPILE.md or
  * disable the test if `file` is not available
.

Reply:
I really think `file` should be part of every Unix system. The `file` utility is also [part of the POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/file.html). If we really are that picky about this stuff, then we would also have to mention

- `awk`,
- `bash`
- `grep`,
- `find` and
- `sed`

as dependencies of Elektra.

> add `file` as a dependency to COMPILE.md

We do not need the `file` utility to **compile** Elektra.

Reply:
then we should probably list those deps for the test helpers in compile.md and explicitly install them in the images to reflect that.

Reply:
> then we should probably list those deps for the test helpers in compile.md

I added the above tools to `Testing.md`. Hope that works for you.

Reply:
Thank you for the hints and the discussion! You already found a good solution.